docs/bit-docs/plugins.md

```diff
@@ -1,5 +1,5 @@
-@page plugins Plugins
-@parent BitDocs 0
+@page plugins API
```

Comment:
@chasenlehara I'm actually waiting on @justinbmeyer to get back before making that change; you can see the git commit message body for more information on why -- I think he mentioned calling this something else like "core".